docs/Files/UI/Solution-Manager/Library/Managing-Global-Properties.md

```diff
@@ -0,0 +1,13 @@
+---
+lang: en-us
+title: Managing Global Properties
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Overview
```

Comment:
The page title should match the title (line 3) and reflect the filename.

So, instead of Overview, it should be Managing Global Properties

Reply:
fixed to use "Managing Global Properties"